cybench/datasets/modified_dataset.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+
+from cybench.datasets.dataset import Dataset
+
+
+class ModifiedTargetsDataset(Dataset):
```

Comment:
This is because only some models (i.e. residual models) will need modified targets. We don't want to modify the original dataset object passed to other models as well.

Reply:
More or less what this new class is doing.